tests/python/test_field.py

```diff
@@ -31,7 +31,7 @@ def test_scalar_field(dtype, shape):
 @pytest.mark.parametrize('shape', field_shapes)
 @ti.host_arch_only
 def test_vector_field(n, dtype, shape):
-    x = ti.Vector.field(n, dtype=dtype, shape=shape)
+    x = ti.Vector.field(n, dt=dtype, shape=shape)
```

Comment:
Let's edit it in `ti.Matrix._Vector_field` to permently fix the issue.

Reply:
Sounds good! Please feel free to push to this PR (or directly open a different PR and close this if you want).